zeppelin-interpreter/src/main/java/org/apache/zeppelin/dep/TransferListener.java

```diff
@@ -122,7 +120,7 @@ public void transferSucceeded(TransferEvent event) {
   @Override
   public void transferFailed(TransferEvent event) {
     transferCompleted(event);
-    event.getException().printStackTrace(out);
+    logger.warn(event.getException().getMessage());
```

Comment:
I'd convert to logger.warn(message, event.getException()) here
https://www.slf4j.org/api/org/slf4j/Logger.html#warn(java.lang.String,%20java.lang.Throwable)